internal/cli/filtering.go

```diff
@@ -280,7 +280,9 @@ func parseDayRange(date string) (start time.Time, end time.Time, err error) {
 		return
 	}
 
-	start = startOfDay(day)
+    // we add -1 second so that the day range ends at 23:59:59
+    // i.e, the 'new day' begins at 00:00:00
+	start = startOfDay(day).Add(time.Second * -1)
```

Comment:
What was the value of start before? `00:00:00` or `00:00:01` ?
If start was already `00:00:00` then it seems that subtracting a second should be done to from `end`.

Reply:
Here's the test: https://github.com/zk-org/zk/pull/385